### ISSUE TITLE
CDWHCITransferStageData Trying to allocate more than 16 instances

### ISSUE THREAD

**Issue by hinxx** [rsta2/circle#147]
Juggling the mice while running sample/10-usbmouse causes the following to be printed in serial console:
```
00:01:21.31 CDWHCITransferStageData: Trying to allocate more than 16 instances
```
Afterwards the app becomes unresponsive.

**Comment by rsta2** [comment]
Have you plug-and-play removed and re-attached the mice, before this happens?

**Comment by hinxx** [comment]
Yes, I was doing this quite a lot. I would think about 16 times, as the number in the message says. 15 times with no issue, and the message only appeared at the last re-attachment.

**Comment by rsta2** [comment]
I think, I know the reason for the problem and I can fix it. Will do so soon.

**Comment by rsta2** [comment]
The issue has been fixed on the *develop* branch.

**Comment by hinxx** [comment]
After testing this it looks like the problem is solved; I do not get `CDWHCITransferStageData: Trying to allocate more than 16 instances` message any more, even after reconnecting the mouse >16 times.

**Comment by hinxx** [comment]
There seems to be a regression introduced compared to the master branch.
If I remove/re-attach the mouse quickly the `10-usbmouse` sample from the develop branch locks up and a reboot is required.
This is not the case with the master branch.

**Comment by rsta2** [comment]
Thanks for testing. I'm afraid, I will not be able to handle very quick USB device attachments / removes in a row. With the latest fix from issue #146 it may be better, but not perfect.

**Comment by hinxx** [comment]
OK, I have not big issues with the fact that super fast re-attachments may not work.
